torchvision/models/detection/backbone_utils.py

```diff
@@ -197,7 +203,10 @@ def _mobilenet_extractor(
 
         if returned_layers is None:
             returned_layers = [num_stages - 2, num_stages - 1]
-        assert min(returned_layers) >= 0 and max(returned_layers) < num_stages
+        if min(returned_layers) <= 0 or max(returned_layers) >= num_stages:
```

Comment:
Bug:
```suggestion
        if min(returned_layers) < 0 or max(returned_layers) >= num_stages:
```

Reply:
It is as per current logic: `min(returned_layers) == 0` is not erroneous: https://github.com/pytorch/vision/blob/59c723cb45d0f8ab897cc7836d408e9fdde4b552/torchvision/models/detection/backbone_utils.py#L200